modules/appeals_api/app/sidekiq/appeals_api/appeal_received_job.rb

```diff
@@ -9,152 +9,71 @@ class AppealReceivedJob
-
-      vanotify_service.send_email(params({ template_id: }))
-      StatsD.increment(STATSD_CLAIMANT_EMAIL_SENT, tags: { appeal_type: 'sc', claimant_type: })
-    end
-
-    private
-
-    attr_accessor :opts
-
-    def vanotify_service
-      @vanotify_service ||= VaNotify::Service.new(Settings.vanotify.services.lighthouse.api_key)
-    end
-
-    def params(template_opts)
-      [
-        lookup,
-        template_opts,
-        personalisation
-      ].reduce(&:merge)
-    end
-
-    def lookup
-      return { email_address: opts['claimant_email'] } if opts['claimant_email'].present?
-
-      if opts['email_identifier']['id_type'] == 'email'
-        { email_address: opts['email_identifier']['id_value'] }
-      else
-        { recipient_identifier: { id_value: opts['email_identifier']['id_value'],
-                                  id_type: opts['email_identifier']['id_type'] } }
+      if appeal_id.blank? || appeal_class_str.blank? || date_submitted_str.blank?
+        argument_list = [appeal_id, appeal_class_str, date_submitted_str]
+        raise "Missing arguments in #{self.class.name}: Received #{argument_list.join(', ')}"
       end
-    end
 
-    def template_id(template)
-      t = claimant? ? "#{template}_claimant" : template
-      template_id = Settings.vanotify.services.lighthouse.template_id.public_send(t)
+      appeal = appeal_class_str.constantize.find(appeal_id)
 
-      [t, template_id]
-    end
-
-    def personalisation
-      p = { 'date_submitted' => date_submitted }
-      if claimant?
-        p['first_name'] = opts['claimant_first_name']
-        p['veterans_name'] = opts['first_name']
-      else
-        p['first_name'] = opts['first_name']
+      unless appeal.form_data.present? && appeal.auth_headers.present?
+        raise "Missing PII for #{appeal_class_str} #{appeal_id}"
       end
-      { personalisation: p }
-    end
-
-    def log_error(guid, type)
-      Rails.logger.error "No lookup value present for AppealsApi::AppealReceived notification #{type} - GUID: #{guid}"
-    end
 
-    def guid
-      opts['guid']
-    end
-
-    def date_submitted
-      @date_submitted ||= DateTime.iso8601(opts['date_submitted']).strftime('%B %d, %Y')
-    end
-
-    def valid_email_identifier?
-      if claimant?
-        opts['claimant_email'].present?
+      vanotify_service.send_email(vanotify_args(appeal, date_submitted_str))
+
+      StatsD.increment(
+        STATSD_CLAIMANT_EMAIL_SENT,
+        tags: {
+          appeal_type: appeal.class.name.demodulize.scan(/\p{Upper}/).map(&:downcase).join,
+          claimant_type: appeal.claimant.signing_appellant? ? 'non-veteran' : 'veteran'
+        }
+      )
+    end
+
+    def vanotify_args(appeal, date_submitted_str)
+      common_info = {
+        template_id: appeal_template_id(appeal),
+        date_submitted: DateTime.iso8601(date_submitted_str).strftime('%B %d, %Y')
+      }
+
+      if appeal.claimant.signing_appellant?
```

Comment:
My understanding is that this (`appeal.claimant.signing_appellant?`) is the way to check whether the appeal has a non-veteran claimant (whether the claimant is the signer)